packages/turf-bezier-spline/lib/spline.ts

```diff
@@ -194,3 +194,6 @@ function B(t: number) {
     (1 - t) * (1 - t) * (1 - t),
   ];
 }
+
+export { Spline, Point };
+export default Spline;
```

Comment:
We might consider marking these all as @deprecated (in another PR) if we're intending to remove them eventually